### PR TITLE
Fix the focus of the node e2e serial job

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -11606,7 +11606,7 @@
       "--node-test-args=--feature-gates=DynamicKubeletConfig=true,PodPriority=true,LocalStorageCapacityIsolation=true --kubelet-flags=\"--cgroups-per-qos=true --cgroup-root=/\"",
       "--node-tests=true",
       "--provider=gce",
-      "--test_args=--nodes=1 --focus=\"\\[Serial\\]|\\[NodeFeature:.+\\]\" --skip=\"\\[Flaky\\]\"",
```
Comment (yujuhong) [inline jobs/config.json]: What I actually wanted to do is to enforce `[Serial]` `AND` `[NodeFeature:.+]`.
Is there an easy way to express that? /cc @krzyzacy

Comment (krzyzacy) [reply jobs/config.json]: regex is hard :-)

Comment (yujuhong) [reply jobs/config.json]: Does calling `--focus` twice work (I'd assume not...)?

Comment (krzyzacy) [reply jobs/config.json]: I don't think so..

Comment (krzyzacy) [reply jobs/config.json]: from stackoverflow: https://regex101.com/r/iS5jJ3/1, but not sure what golang takes

Comment (yujuhong) [reply jobs/config.json]: /shrug

I'll fix the test job with this PR first, and look for a way to express this better later

**Pull request merged**